src/bp/common/typings.ts

```diff
@@ -188,7 +188,7 @@ export interface ServerHealth {
 }
 
 export interface BotHealth {
-  status: 'mounted' | 'unmounted' | 'disabled' | 'error'
+  status: 'healthy' | 'unhealthy' | 'unmounted' | 'disabled'
```

Comment:
what's the difference between unmounted and disabled ?

Reply:
Technically it could be unmounted without being disabled, but you're right that it's a bit over-complicated for nothing. I'll remove it